app/routes/_auth.certificate._index.tsx

```diff
@@ -115,6 +115,7 @@ export default function CertificateIndexRoute() {
       <Loading
         img={pendingSvg}
         desc="We have received your request, and will notify you when your certificate is ready"
+        alt="two-people-processing"
```

Comment:
Don't use `-`, it's supposed to be natural language.  Maybe, "Two people at a computer processing a document together"

Reply:
oh so it's literally to describe them alright I'll keep that in mind